react/Autosuggest/Autosuggest.js

```diff
@@ -5,9 +5,16 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import ReactAutosuggest from 'react-autosuggest';
 import IsolatedScroll from 'react-isolated-scroll';
-import omit from 'lodash.omit';
+
+import get from 'lodash/get';
+import invoke from 'lodash/invoke';
+import omit from 'lodash/omit';
 
 import TextField from '../TextField/TextField';
+import smoothScroll from '../SmoothScroll/SmoothScroll';
```

Comment:
But I need to use it for classifications too. So I'll probably just move it to /utils/smoothScroll